docs: link updateTag from cacheTag reference (#92379)
vercel/next.js: docs/01-app/03-api-reference/04-functions/cacheTag.mdx

Reviewer: I think we should target this section rather

Use `updateTag` first, within a Server Function/Action - to get read your writes behavior, or use `revalidateTag` if you delay in the revalidation is acceptable, with stale-while-revalidate mechanics

}
```

> You can then purge the cache on-demand from a [Server Action](/docs/app/getting-started/mutating-data) or [route handler](/docs/app/api-reference/file-conventions/route):

Author: ```suggestion
You can then purge the cache on-demand from a [Server Function](/docs/app/getting-started/mutating-data) or [route handler](/docs/app/api-reference/file-conventions/route):
```

> You can then purge the cache on-demand from a [Server Function](/docs/app/getting-started/mutating-data) or [route handler](/docs/app/api-reference/file-conventions/route):

Author: ```suggestion
You can then purge the cache on-demand from a [Server Function](/docs/app/getting-started/mutating-data) or [Route Handler](/docs/app/api-reference/file-conventions/route):
```

> - Use [`updateTag`](/docs/app/api-reference/functions/updateTag) inside a Server Action for immediate expiry so the next read fetches fresh data (read-your-own-writes).

Reviewer: ```suggestion
- Use [`updateTag`](/docs/app/api-reference/functions/updateTag) inside a Server Action for read-your-own-writes scenarios, where a user makes a change and the next read should fetch fresh data immediately.
```

wdyt?

Author: Yep thats better

> - Use [`updateTag`](/docs/app/api-reference/functions/updateTag) inside a Server Action for read-your-own-writes scenarios, where a user makes a change and the next read should fetch fresh data immediately.

Author: ```suggestion
- Use [`updateTag`](/docs/app/api-reference/functions/updateTag) inside a Server Function for read-your-own-writes scenarios, where a user makes a change and the next read should fetch fresh data immediately.
```

> - Use [`revalidateTag`](/docs/app/api-reference/functions/revalidateTag) when a stale-while-revalidate delay is acceptable, or when invalidating from a route handler.

Reviewer: ```suggestion
- Use [`revalidateTag`](/docs/app/api-reference/functions/revalidateTag) when it is acceptable to serve stale data while revalidation happens in the background, or when revalidating from a route handler.
```
